### ISSUE TITLE
Pin log tabs

### ISSUE THREAD

**Issue by olensmar** [kubeshop/monokle#2174]
I would like to "pin" the Logs tab for certain pods to a dedicated area in the UI so I can continue monitoring those logs while I am selecting other resources / doing other things. 

I understand that if I disconnect from my cluster these Logs tabs would be closed - but perhaps in the future it would be nice to save and reconnect them when I connect to my cluster again!?

**Comment by bss-dmitry-shmakov** [comment]
yeah that would be neat. or at least expand the logs tab to full width, it looks miserable without word wrap, cannot use... and very surprised that this was not done until now, since it renders product feature unusable (nobody will read logs through that interface since most containers output very long lines with the cri-o logging format nowadays).  maybe someone could point us to the code so we can patch it and PR to the project?
